.travis.yml

```diff
@@ -9,14 +9,17 @@ matrix:
   include:
     - python: 3.6
       env: CI_TARGET=checkqa TOXENV=checkqa
+    - python: pypy
```

Comment:
could this be moved abouve `checkqa`?  Would be nice to keep all test suite runs grouped together, and the lint stuff at the bottom.